projects/aws/eks-charts/aws-for-fluent-bit/Makefile

```diff
@@ -0,0 +1,42 @@
+AWS_FOR_FLUENT_BIT_CHART_REPO_URL?=https://github.com/aws/eks-charts/
+AWS_FOR_FLUENT_BIT_CHART_REPO?=eks-charts
```

Comment:
Can you declare the repo name first and then reference the value in the URL like [here](https://github.com/aws/eks-distro-build-tooling/blob/main/projects/prometheus-community/helm-charts/Makefile#L1-L2)? Also can you add `.git` at the end of the clone URI?